src/engine/test/source/builder/builders/stageBuilderCheck_test.cpp

```diff
@@ -100,24 +100,151 @@ TEST(StageBuilderCheckTest, ListBuildsCorrectExpression)
+                                    "Check stage: The \">\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field>[\"value1\",\"value2\"]")"},
+                                    "Check stage: The \">\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field>false")"},
+                                    "Check stage: The \">\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field>null")"},
+                                    "Check stage: The \">\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field<{\"key\":\"value\"}")"},
+                                    "Check stage: The \"<\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field<[\"value1\",\"value2\"]")"},
+                                    "Check stage: The \"<\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field<false")"},
+                                    "Check stage: The \"<\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field<null")"},
+                                    "Check stage: The \"<\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field<={\"key\":\"value\"}")"},
+                                    "Check stage: The \"<=\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field<=[\"value1\",\"value2\"]")"},
+                                    "Check stage: The \"<=\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field<=false")"},
+                                    "Check stage: The \"<=\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field<=null")"},
+                                    "Check stage: The \"<=\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field>={\"key\":\"value\"}")"},
+                                    "Check stage: The \">=\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field>=[\"value1\",\"value2\"]")"},
+                                    "Check stage: The \">=\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field>=false")"},
+                                    "Check stage: The \">=\" operator only allows operate with numbers or string"),
+                    std::make_tuple(Json {R"("field>=null")"},
+                                    "Check stage: The \">=\" operator only allows operate with numbers or string")));
```

Comment:
Are we sure Check stage all cases in Testing::values map to the same fmt string "`Check stage: The \">=\" operator only allows operate with numbers or string""`

Reply:
Disregard. Clear now.